docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/paragraph--cgov-card-internal.html.twig

```diff
@@ -6,6 +6,12 @@
  # we will duplicate it.
  #}
 {% if content.field_featured_item[0] %}
+
+  {% set cardType = 'featureCard' %}
+  {% if 'thumbnail' in _context.theme_hook_suggestions[2] %}
```

Comment:
How solid it this approach? Random array indexes could be problematic. The theme hooks can change. Is there anyway to get at the display mode?

Reply:
switched to use `_context.view_mode` which seems like a much more stable attribute